submissions/m-ruslan/js-dom/app.js

```diff
@@ -0,0 +1,86 @@
+document.addEventListener("DOMContentLoaded", () => {
+  console.log("DOM is loaded");
+  loadPageData("info");
+
+  for (i in pageSections) {
+    document
+      .querySelector("#" + pageSections[i].id)
+      .addEventListener("click", (event) => {
```

Comment:
I suggest to user [event delegation](https://learn.javascript.ru/event-delegation) instead of adding a listener on every menu item here. Let me know if you need more detailed explanation

Reply:
Thank you.
I've changed code for adding event delegation. Could you please check?